src/Telegram.Bot/Types/ChatPhoto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
You don't need
`using System;
using System.Collections.Generic;
using System.Text;`
for this class